services/crypto_get_info.proto

```diff
@@ -59,6 +59,64 @@ message CryptoGetInfoResponse {
      */
     ResponseHeader header = 1;
 
+    message CryptoAllowance {
```

Comment:
This can be replaced with `CryptoApproval`. We might consider renaming `CryptoApproval` with `CryptoAllowance` and just letting the `approve` action be in the service name and field names.

Reply:
Agreed, can reuse `basic_types`

Reply:
Can potentially use composition to consolidate `TokenApproval` and `TokenAllowance`.

Reply:
Elected to re-use the basic types + rename them to `*Allowances`. The oneof in the `NftAllowances` is also being removed since oneof doesn't support repeated (funny, only saw the error after the packed was removed).